project_addon/project_addon/doctype/timesheet_record/timesheet_record.py

```diff
@@ -40,14 +19,14 @@ def create_timesheet(self):
 		description = "{0} : {1}".format(self.goal, self.result)
 		actual_hours = round(float(self.actual_time) / 3600, 6)
 		timesheet = frappe.new_doc("Timesheet")
-
-		timesheet.update({
-				"project": self.project,
+		timesheet.update(
+			{
+				"parent_project": self.project,
```

Comment:
In Timesheet Doctype 'parent_project' field is some new field introduced in latest versions; i guess a replacement of 'project' field. But in Existing phamos.eu when have 'project' field to map 'Timesheet Record' and 'Timesheet' project.

Reply:
the `parent_project`is as far as I know a erpnext standard field. I added `project`to Timesheet in our system to make sure, that that backlinks and other references work as we are used to using them... let me know if there is any ttrouble or confusion with that.

Reply:
Yes sorry for confusion we discussed this field earlier, so we need to use 'project' instead of 'parent_project' field, so this should be corrected in PR @KanchanChauhan